system_test/aest_commands_SUITE.erl

```diff
@@ -45,8 +45,10 @@ init_per_testcase(_TC, Config) ->
     aest_nodes:ct_setup(Config).
 
 end_per_testcase(_TC, Config) ->
-    aest_nodes:ct_cleanup(Config).
-
+    try aest_nodes:ct_cleanup(Config)
```

Comment:
We could move the try catch into `ct_cleanup` since the function is intended to be run from Common Test anyway

Reply:
I was unsure if it was used differently. Will do